pkg/lib/localstore/ftsearch/ftsearch.go

```diff
@@ -133,6 +133,9 @@ func (f *ftSearch) BatchIndex(docs []SearchDoc) (err error) {
 		}
 	}()
 	for _, doc := range docs {
+		if ctx.Err() == context.Canceled {
```

Comment:
I suppose, should be `error.Is`

Reply:
for the context error it makes no sense because it has the defined finite list of constant errors: context.Canceled or context.DeadlineExceeded